teacher_student_dynamics/runners/hmm_multi_teacher_runner.py

```diff
@@ -687,7 +688,7 @@ def _setup_data(
                     activation=config.activation,
                     zero_matrix=zero_matrix,
                     rotation_matrix=rotation_matrix,
-                    feature_matrix=F1_tilde,
+                    feature_matrix=F1,
```

Comment:
I think this should stay F_1 tilde at this point since it is a day d matrix

Reply:
?

Reply:
@seblee97 has it changed how we implement this that we no longer do the zero padding and rotation in this function?

Reply:
Nothing has really changed on this bit I just changed the names.

Reply:
The notation F1 is when the padding and zero is done, while F1_tilde has a DbyD size. I think at this point the padding and rotation has not been done.  The variable you renamed F1 should still be F1_tilde

Reply:
ok looking again, now I'm super confused. Maybe we actually don't at all need what is currently defined as F1 (and associated variables for later task F's below). We just need to feed in np.sqrt(self._input_dimension / self._latent_dimension) * F_tilde_part to the argument for feature_matrix in the hidden mainfold constructor. Maybe this is what you are saying clem. Because we actually do the zero padding and rotation in the hidden manifold class initialisation. What do you think?

Reply:
yes

Reply:
tho we need need to keep the square root of D since the square root of the eigenvalues was already there.